python_modules/event-pipeline-demo/event_pipeline_demo/configs_spark.py

```diff
@@ -0,0 +1,1686 @@
+'''NOTE: THIS FILE IS AUTO-GENERATED. DO NOT EDIT
```

Comment:
So this is worth discussing. I thought this would be a "generate once, hand-edit from now on" type file.